app/Importers/BinBinDataImporter.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Importers;
+
+use GuzzleHttp\Exception\GuzzleException;
+use Symfony\Component\DomCrawler\Crawler;
+
+class BinBinDataImporter extends DataImporter
+{
+    protected Crawler $sections;
+
+    public function extract(): static
+    {
+        try {
+            $response = $this->client->get("https://www.binbin.tech/lokasyonlar");
+            $html = $response->getBody()->getContents();
+        } catch (GuzzleException) {
```

Comment:
What do we do with the caught exception?
Maybe we would store that exception as a meta information in import info details or smth like that.

Reply:
this commit was placed here by accident, actual BinBin is in #117